dev/reference/tools/tidb-lightning/deployment.md

```diff
@@ -184,9 +184,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB Lightning package (choose the same version as that of the TiDB cluster):
 
-- **v2.1.9**: <https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz>
-- **v2.0.9**: <https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz>
-- Latest unstable version: <https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz>
+- **v3.0.2**: <https://download.pingcap.org/tidb-toolkit-v3.0.2-linux-amd64.tar.gz>
```

Comment:
I think it's better to maintain this information at [tools download page](https://github.com/pingcap/docs/blob/master/dev/reference/tools/download.md), what about you? @lilin90